python/ray/tune/tests/test_tune_restore.py

```diff
@@ -0,0 +1,61 @@
+# coding: utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import glob
+import os
+import tempfile
+import unittest
+
+import ray
+from ray import tune
+from ray.rllib import _register_all
+
+
+class TuneRestoreTest(unittest.TestCase):
+    def setUp(self):
+        self.curdir = os.path.abspath('.')
```

Comment:
```suggestion
        self.curdir = os.path.abspath(".")
```

This should address https://travis-ci.com/ray-project/ray/jobs/197103865

Reply:
That's great! Could you give me some reference on why single quotes causes error in travis-ci? What's more, are there other mistakes which are easily made, like this one?